src/devices/lixee.ts

```diff
@@ -13,6 +14,65 @@ const ea = exposes.access;
 const e = exposes.presets;
 
 const NS = 'zhc:lixee';
+
+const local = {
+	modernExtend: {
+        addCustomClusterManuSpecificLixee: () =>
+            deviceAddCustomCluster('liXeePrivate', {
```

Comment:
```suggestion
            deviceAddCustomCluster('liXeePrivateCustom', {
```

Could you check if this now work from the dev console? Otherwise there is a conflict with the already existing cluster.

Reply:
Ok thank you for this